src/components/collection-count/collection-count.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { t } from '@lingui/macro';
+import { CollectionAPI, CollectionExcludesType } from 'src/api';
+import { Spinner } from '@patternfly/react-core';
+import { AlertType } from 'src/components';
+import { errorMessage } from 'src/utilities';
+
+interface IProps {
+  distributionPath: string;
+}
+interface IState {
+  collectionCount: number;
+  alerts: AlertType[];
+  loading: boolean;
+}
+
+export class CollectionCount extends React.Component<IProps, IState> {
+  constructor(props) {
+    super(props);
+    this.state = {
+      collectionCount: null,
+      alerts: [],
+      loading: true,
+    };
+  }
+
+  componentDidMount() {
+    this.getCollectionCount(this.props.distributionPath);
+  }
+
+  render() {
+    const { collectionCount, loading } = this.state;
+    return !loading ? <td>{collectionCount}</td> : <Spinner />;
```

Comment:
```suggestion
    return !loading ? <>{collectionCount}</> : <Spinner />;
```

(or a span)

(`CollectionCount` is already rendered within a td, so this would produce a `<td><td>...`)